src/fides/api/ops/util/saas_util.py

```diff
@@ -299,3 +299,9 @@ def encode_file_contents(file_path: str) -> str:
     file_path = load_file([file_path])
     with open(file_path, "rb") as file:
         return bytes_to_b64_str(file.read())
+
+
+def change_string_to_camel_case(s: str) -> str:
```

Comment:
Let's simplify this function name to `to_camel_case`. Can you also add a simple test in `test_saas_util` so we can see the expected behavior?

Reply:
nit: how about renaming it to `to_pascal_case` https://en.wikipedia.org/wiki/Naming_convention_(programming)#Examples_of_multiple-word_identifier_formats